modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -1,28 +1,36 @@
 
-Multi-node or entire-AZ failures (especially in cloud environments), and some forms of human error may result in ‘stuck’ partitions with fewer replicas than required to make a quorum. In such failure scenarios, some data loss may be unavoidable. This description of node-wise partition recovery helps admins understand what they can or cannot recover.
+Multi-node or entire-AZ failures (especially in cloud environments), along with some forms of human error, can result in ‘stuck’ partitions where there are fewer replicas than required to make a quorum. In such failure scenarios, some data loss may be unavoidable. Node-wise partition recovery provides a way to unsafely recover at least a portion of your data using remaining replicas, which are moved off of target nodes and allocated to healthy nodes. In one step, this process repairs partitions while draining the target nodes. This topic helps admins understand what they can or cannot recover using node-wise partition recovery.
 
-IMPORTANT: This is a last-ditch measure when all other recovery options have failed. In some cases, no remaining replicas may exist for the partitions on the dead nodes. This recovery method is intended for scenarios where you are already experiencing data loss - the goal is to stop the loss of additional data.
-
-Node-wise partition recovery allows you to unsafely recover at least a portion of your data using whatever replicas are remaining. All replicas are moved off the target nodes and allocated to healthy nodes. In one step, this process repairs partitions while draining the target nodes.
+IMPORTANT: Only use this operation as a last-resort measure when all other recovery options have failed. In some cases, there may be no remaining replicas for the partitions on the dead nodes. This recovery method is intended for scenarios where you have already experienced data loss, with the goal being to stop the loss of additional data.
 
 == Perform the recovery operation
 
-You perform node-wise partition recovery using the `rpk` command `rpk cluster partitions unsafe-recover`. This command includes an interactive prompt to confirm execution of the generated recovery plan as it is a destructive operation. When you trigger node-wise partition recovery, the partitions on the node are rebuilt in a best-effort basis. As a result of executing this operation, you may lose some data that has not yet been replicated to the surviving partition replicas.
+Launch node-wise partition recovery with `rpk cluster partitions unsafe-recover`. This command includes an interactive prompt to confirm execution of the generated recovery plan, as it is a destructive operation. When you trigger node-wise partition recovery, the partitions on the node are rebuilt on a best-effort basis. As a result of executing this operation, you may lose some data that has not yet been replicated to the surviving partition replicas.
```

Comment:
@Feediver1 its worth expanding here on the RF=1 case, e.g.   'In cases where there are zero surviving partition replicas, such as a topic with replication factor 1, partition recovery rebuilds empty partitions with no data, allowing Producers to continue writing to the partition despite the fact that no data can be recovered in these situations'.

This is more accurate than what's written ('some data' can be recovered)